integration-tests/src/test/kotlin/org/stellar/anchor/platform/test/EventProcessingServerTests.kt

```diff
@@ -0,0 +1,76 @@
+package org.stellar.anchor.platform.test
+
+import org.stellar.anchor.api.event.AnchorEvent
+import org.stellar.anchor.event.EventService.EventQueue.TRANSACTION
+import org.stellar.anchor.platform.TestConfig
+import org.stellar.anchor.platform.config.PropertyEventConfig
+import org.stellar.anchor.platform.event.DefaultEventService
+import org.stellar.anchor.util.GsonUtils
+import org.stellar.anchor.util.Sep1Helper
+
+class EventProcessingServerTests(config: TestConfig, toml: Sep1Helper.TomlContent, jwt: String) {
+  companion object {
+    val eventConfig =
+      GsonUtils.getInstance().fromJson(eventConfigJson, PropertyEventConfig::class.java)!!
+  }
+  fun testOk() {
+    val eventService = DefaultEventService(eventConfig)
+    val session = eventService.createSession("testOk", TRANSACTION)
+    val quoteEvent = GsonUtils.getInstance().fromJson(testQuoteEvent, AnchorEvent::class.java)
```

Comment:
Can this be replaced with `AnchorEvent` constructor? I don't think we need to test deserialization here.

Reply:
We are not testing the deserialization. It is a convenient way to construct from JSON rather in Java. The pattern is used in AP.

Reply:
I would prefer we if stop using this pattern because it's easier to see diffs if it was initialized using Java. We wouldn't need to worry about white space diffs in JSON text blocks.

This is just a nit, so we don't need to block the PR on this.

Reply:
> it's easier to see diffs if it was initialized using Java

I am not sure if I got it correctly. The current approach deserialize JSON into Java object. The assert should diff on the Java objects.